packages/dashmate/configs/getConfigFileMigrationsFactory.js

```diff
@@ -137,6 +137,15 @@ function getConfigFileMigrationsFactory(homeDir, defaultConfigs) {
           });
         return configFile;
       },
+      '0.25.0-dev.22': (configFile) => {
+        Object.entries(configFile.configs)
+          .forEach(([, options]) => {
+            options.platform.dapi.api.docker.image = base.get('platform.dapi.api.docker.image');
```

Comment:
I believe dashcore version handling should be added here, because after succesfull testnet rollout, v25 will become the master branch, but the v20 is hardcoded right now. I think we want to leave v19 for mainnet nodes for now

Reply:
Not sure what you mean, could please elaborate?

Reply:
I think @strophy wants you to move this to "not mainnet condition". I agree with your @strophy approach but at this moment it doesn't actually matter because this versions will be overwritten before we go to mainnet.

Reply:
Ah sweet, Thats what I did. @shumkov could you please review my changes

Reply:
No, you didn't. I mean you should put these three lines under line 144